README.md

```diff
@@ -32,24 +32,24 @@ Note that variant calls are associated with patients, not samples. This is the "
 
 ## Running Locally
 
-Change the `--guacamole-jar` path below for your environment.
+Change the `--guacamole-jar` and `--guacamole-dependencies-jar` paths below for your environment.
 
 ```
 vcb-guacamole-local \
     base_config.json \
     infrastructure-configs/local.json \
     guacamole-configs/default.json \
     benchmarks/local-wgs1/benchmark.json \
-    --guacamole-jar ~/sinai/git/guacamole/target/guacamole-with-dependencies-0.0.1-SNAPSHOT.jar \
+    --guacamole-jar ~/sinai/git/guacamole/target/guacamole-0.0.1-SNAPSHOT.jar \
```

Comment:
in any case I think for the documentation here you should sub in such a var, so `$GUAC_HOME/target/guacamole-…` etc.